### PR TITLE
redirect console root to app URL

### PR DESCRIPTION
**Pull request by apoland**
- depends on change in https://github.com/aserto-dev/service-host/pull/12

### CODE REVIEW

File: pkg/app/topaz.go [hunk 1]
```diff
@@ -164,6 +164,7 @@ func (e *Topaz) ConfigServices() error {
 
 			consoleConfig := con.(*ConsoleService).PrepareConfig(e.Configuration)
 			if contains(serviceConfig.registeredServices, "console") {
```
Comment (gertd) [inline pkg/app/topaz.go:166]: Results in a panic
```
panic: http: multiple registrations for /

goroutine 1 [running]:
net/http.(*ServeMux).Handle(0x14000d1c580, {0x107486670, 0x1}, {0x107916860, 0x1400099bce0})
	/Users/gertd/.gobrew/versions/1.20.14/go/src/net/http/server.go:2516 +0x384
github.com/aserto-dev/topaz/pkg/app.(*Topaz).ConfigServices(0x140007848c0)
	/Users/gertd/workspace/cur/topaz/pkg/app/topaz.go:167 +0x940
main.glob..func2(0x10864d2a0, {0x1400068ee60, 0x0, 0x2})
	/Users/gertd/workspace/cur/topaz/cmd/topazd/topaz_run.go:51 +0x178
github.com/spf13/cobra.(*Command).execute(0x10864d2a0, {0x1400068ee40, 0x2, 0x2})
	/Users/gertd/workspace/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0xb78
github.com/spf13/cobra.(*Command).ExecuteC(0x10864cce0)
	/Users/gertd/workspace/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x798
github.com/spf13/cobra.(*Command).Execute(0x10864cce0)
	/Users/gertd/workspace/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039 +0x30
main.main()
	/Users/gertd/workspace/cur/topaz/cmd/topazd/main.go:31 +0x78
```

**Review by gertd — CHANGES_REQUESTED** [review]
This solution results in a panic

File: pkg/app/topaz.go (hunk 1 above)
Comment (apoland) [reply pkg/app/topaz.go:166]: @gertd this depends on the change in https://github.com/aserto-dev/service-host/pull/12

Comment (gertd) [reply pkg/app/topaz.go:166]: Can you add that to the PR by including the go.mod changes require to update the dependency?

File: go.mod [hunk 2]
```diff
@@ -9,7 +9,7 @@ go 1.20
 // replace github.com/aserto-dev/azm => ../azm
 
 // replace github.com/aserto-dev/go-topaz-ui => ../go-topaz-ui
-// replace github.com/aserto-dev/service-host => ../service-host
+//replace github.com/aserto-dev/service-host => ../service-host
```
Comment (gertd) [inline go.mod:12]: Please remove go.mod from PR as there are no changes

**Review by gertd — CHANGES_REQUESTED** [review]
Looks, good, works great, only one small request regarding the go.mod file

File: go.mod (hunk 2 above)
Comment (BogdanIrimie) [reply go.mod]: Removed the change in go.mod

**Review by gertd — APPROVED** [review]

**Pull request merged**